### PR TITLE
[doc] remove dead links in README.md

### PR DESCRIPTION
**Pull request by extract**
After commit #3462 the links in the README.md broke in the Common Tools section. This commit removes those links as they seem to no longer be necessary.
@csordasmarton

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -195,10 +195,8 @@ For details see
 Useful tools that can also be used outside CodeChecker.
 
 * [Build Logger (to generate JSON Compilation Database from your builds)](/analyzer/tools/build-logger/README.md)
-* [Plist to HTML converter (to generate HTML files from the given plist files)](/docs/tools/plist_to_html.md)
 * [Report Converter Tool (to convert analysis results from other analyzers to the codechecker report directory format))](/docs/tools/report-converter.md)
 * [Translation Unit Collector (to collect source files of a translation unit or to get source files which depend on the given header files)](/docs/tools/tu_collector.md)
-* [Report Hash generator (to generate unique hash identifiers for reports)](/docs/tools/codechecker_report_hash.md)
```
Comment (csordasmarton) [inline docs/README.md:201]: The documentation is moved to here: https://github.com/Ericsson/codechecker/blob/master/docs/tools/report-converter.md#report-hash-generation-module

File: docs/README.md [hunk 2]
```diff
@@ -195,10 +195,8 @@ For details see
 Useful tools that can also be used outside CodeChecker.
 
 * [Build Logger (to generate JSON Compilation Database from your builds)](/analyzer/tools/build-logger/README.md)
-* [Plist to HTML converter (to generate HTML files from the given plist files)](/docs/tools/plist_to_html.md)
```
Comment (csordasmarton) [inline docs/README.md:198]: The documentation is moved to here: https://github.com/Ericsson/codechecker/blob/master/docs/tools/report-converter.md#plist-to-html-tool

**Review by csordasmarton — DISMISSED** [review]
These links are not removed but the documentations were moved to a different place.

**Comment by bruntib** [comment]
@extract Thanks for the fix. However, the pointed documentations are not missing, but moved to other locations as @csordasmarton mentioned. Could you please fix these instead of removing them? Thank you!

**Comment by extract** [comment]
hi

Sorry took some time, it should now be correct. I also fixed in report-converter.md as the table of content was linking incorrectly.

**Comment by bruntib** [comment]
Thank you very much for the contribution. I'll merge it soon.

**Review by bruntib — APPROVED** [review]

**Pull request merged**